regression/cbmc-viewer/test.py

```diff
@@ -253,6 +254,21 @@ def main():
 
     set_up_loggers(args)
 
+    # Path to root of cbmc source tree
+    source_root = os.path.abspath(
+        os.path.join(os.path.dirname(__file__) or '.', "..", "..")
```

Comment:
Is there a reason for duplicating `".."`?

Reply:
The regression/cbmc-viewer directory is two levels below the top-level cbmc source tree root.  I could have written "../.." instead of ".." "..".